website/package.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
I wasn't sure how I'm supposed to install this - given the monorepo structure and all. So I've added locally `"website"` to workspaces here:
https://github.com/redux-saga/redux-saga/blob/b3664a10d3421e2445b596159195e947fc365cc4/package.json#L3

Seems like the best approach for it, WDYT?

Reply:
I don't think adding anything to the workspaces is necessary here because the website is completely independent from the rest of the repo in terms of files and modules. As far as I know (correct me if I'm wrong), including it as a workspace would make it available to the rest of the repo, which has no use.

Reply:
Yes, you are right - I was mainly thinking about the installation part of this. I guess we can just have an additional `yarn.lock` in the `website` directory that would only be installed when working on the docs.